code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1193,7 +1193,9 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 			target.gib(FALSE)
 		if(ADMIN_PUNISHMENT_BSA)
 			bluespace_artillery(target)
-
+		if(ADMIN_PUNISHMENT_FIREBALL)
+			new /obj/effect/overlay/temp/target(get_turf(target))

```

Comment:
this isn't a type

Reply:
/obj/effect/temp_visual/target

Reply:
Old code!

Reply:
@Armhulen this is used by drakes you're going to have to change it.

Reply:
explain

Reply:
oooh it don't work